virttest/guest_agent.py

```diff
@@ -725,3 +725,42 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def openfile(self, args):
+        """
+        :params args: specify file's 'path' and 'mode' option with args.
+        :path: /tmp/agentfile
+        :mode: "w+",if file doesn't exist,mode w+ will create a new file
+        """
```

Comment:
docstring error, no params path and mode

Reply:
@suqinhuang Thanks for your review.  I have updated docstring in 4 functions.